barretenberg/cpp/src/barretenberg/vm/avm_trace/gadgets/avm_ecc.cpp

```diff
@@ -0,0 +1,35 @@
+
+#include "barretenberg/vm/avm_trace/gadgets/avm_ecc.hpp"
+#include "barretenberg/vm/avm_trace/avm_common.hpp"
+
+namespace bb::avm_trace {
+
```

Comment:
feel free to do sth like `using element = grumpkin::g1::affine_element` inside the cpp file.